board/piksiv3/rootfs-overlay/etc/monitrc

```diff
@@ -34,6 +34,7 @@ check process sbp_fileio_daemon_external with pidfile /var/run/sbp_fileio_daemon
 check process piksi_system_daemon with pidfile /var/run/piksi_system_daemon.pid
     start program = "/etc/init.d/S82piksi_system_daemon start"
     stop program = "/etc/init.d/S82piksi_system_daemon stop"
+    if 1 restarts within 3 cycles then exec "/sbin/reboot -f"
```

Comment:
How long is a cycle?  This seems pretty nuclear.   Is there a way to kill everything and relaunch init instead of rebooting?  Would that be faster?

Reply:
the problem is that when piksi_system_daemon restarts, new adapters are spawned, therefore we end up with a bunch of duplicate messages on each adapter (uart0, uart1, etc.)

The "cycles" is unclear to me, but since I wanted to reboot the system after the app dies, I figured that any number would work there (could be 1 restarts within 1 cycle and still do the same)

Reply:
maybe there is a way to kill all the processes with ppid of the just-dead piksi_system_daemon. I'll look into that